src/status_im/ui/screens/chat/message/message.cljs

```diff
@@ -331,21 +368,39 @@
 (def message-height-px 200)
 (def max-message-height-px 150)
 
-(defn on-long-press-fn [on-long-press message content]
-  (on-long-press
-   (concat
-    (when (:show-input? message)
-      [{:on-press #(re-frame/dispatch [:chat.ui/reply-to-message message])
-        :label    (i18n/label :t/message-reply)}])
-    [{:on-press #(react/copy-to-clipboard
-                  (components.reply/get-quoted-text-with-mentions
-                   (get content :parsed-text)))
-      :label    (i18n/label :t/sharing-copy-to-clipboard)}])))
+(defn on-long-press-fn [on-long-press {:keys [public? pinned chat-id] :as message} content]
+  (let [{:keys [group-chat community-id admins]} @(re-frame/subscribe [:chats/current-chat-chat-view])
+        current-pk @(re-frame/subscribe [:multiaccount/public-key])
```

Comment:
this looks duplicated with emoji message

Reply:
Added separate functions to share this logic